examples/simple_regional_confidential_private_beta/main.tf

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Thank you for adding a test! However, we're trying to avoid too many examples. Could you instead consolidate this by adding confidential notes to one of the existing examples/tests?